editor/code/src/client.js

```diff
@@ -3,36 +3,40 @@
 const vscode = require("vscode");
 const vscc = require("vscode-languageclient");
 
-function activate (context) {
-    vscode.window.showInformationMessage('Going to activate!');
+function activate(context) {
+    vscode.window.showInformationMessage('Starting coq-lsp!');
     let clientOptions = {
         documentSelector: [
-            {scheme: 'file', language: 'coq'}
+            { scheme: 'file', language: 'coq' }
         ]
     };
 
     let client = null;
 
-    const restart = () => {
+    const start = () => {
```

Comment:
This really does restart, so the old name was better.

Reply:
I thought it was silly for it to restart as the first thing it does.